RibonDAO/interface
# fix impact description on blocked modal and confirm donation

# Description

- fix impact description on blocked modal and confirm donation page

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#001]()

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Go to modal without tickets and confirm donation page and verify if the impact description is the same of causes page
